app/helpers/hosts_helper.rb

```diff
@@ -492,6 +492,7 @@ def build_error_link(type, id)
   end
 
   def inherited_by_default?(field, host)
+    return true if @force_inherited_params
     return false unless host.hostgroup && host.hostgroup_id_was.nil?
```

Comment:
This gives me an error when trying to view a hostgroup now. 
I changed it to:
(return false unless host.hostgroup && host.hostgroup_id_was.nil?) unless host.nil?
which lets me view the hostgroup, but I've no idea if it's breaking something else.